kytos/utils/napps.py

```diff
@@ -466,10 +466,13 @@ def build_napp_package(napp_name):
 
         """
         files = []
-        path = '/'.join(os.path.abspath(__file__).split('/')[:-1])+'/'
+        path = os.getcwd()
+
         for dir_file in os.walk(path):
             files.extend([dir_file[0] + '/' + file for file in dir_file[2]])
 
+        files = list(filter(lambda x: napp_name in x, files))
```

Comment:
Please add code comments so that we can better understand what are the intentions behind these path transformations.

Reply:
I added the comments, if you have any suggestions I can try to get the code more clear.